demo/spec/components/previews/section_links_preview/with_additional_content.html.haml

```diff
@@ -1,3 +1,4 @@
 = render CitizensAdviceComponents::SectionLinks.new(title: "Related Content", section_title: "Applying to the EU settlement scheme", section_title_url: "/immigration#h-applying-to-the-eu-settlement-scheme") do |c|
   - c.with_section_links(section_links)
-  .example-links-block= link_to "Adviser resources - EU Settlement Scheme", "/advisernet/immigration/staying-in-the-uk/settled-status/adviser-resources-eu-settlement-scheme"
+  - c.with_custom_content do
```

Comment:
This was causing a deprecation warning as it was still using the old format for custom content.